src/ctap/mod.rs

```diff
@@ -3880,13 +3880,19 @@ mod test {
             response,
             Ok(ResponseData::AuthenticatorGetInfo(_))
         ));
-        let response = ctap_state.process_parsed_command(
-            &mut env,
-            Command::AuthenticatorGetInfo,
-            VENDOR_CHANNEL,
-            CtapInstant::new(0),
-        );
-        assert_eq!(response, Err(Ctap2StatusCode::CTAP1_ERR_INVALID_COMMAND));
```

Comment:
I think the intent here was to have a negative test as well, maybe we should use something like Reset here instead to check that some commands are rejected on the vendor HID channels?

Reply:
Good point.

I've renamed this test and add a new test based on the old one that tests for a FIDO command that should not be supported on the Vendor HID.